### PR TITLE
SAR-7090 Add validation to Box 5 page

### PR DESCRIPTION
**Pull request by EviNikolaidou**

### CODE REVIEW

File: it/uk/gov/hmrc/claimvatenrolmentfrontend/controllers/CaptureBox5FigureControllerISpec.scala [hunk 1]
```diff
@@ -23,13 +23,48 @@ class CaptureBox5FigureControllerISpec extends ComponentSpecHelper with CaptureB
...
+      )
+
+      result.status mustBe BAD_REQUEST
+    }
+
+    "return a BAD_REQUEST if the box 5 figure has more than 14 digits" in {
+      lazy val result = post("/box-5-figure")(
+        "box5Figure" -> "0123456789012345"
+      )
+
```
Comment (r-melvin) [inline it/uk/gov/hmrc/claimvatenrolmentfrontend/controllers/CaptureBox5FigureControllerISpec.scala:64]: Can you add a test to cover the negative value?

File: it/uk/gov/hmrc/claimvatenrolmentfrontend/controllers/CaptureBox5FigureControllerISpec.scala [hunk 2]
```diff
@@ -23,13 +23,66 @@ class CaptureBox5FigureControllerISpec extends ComponentSpecHelper with CaptureB
...
+    "return a BAD_REQUEST if the box 5 figure has invalid characters" in {
+      lazy val result = post("/box-5-figure")(
+        "box5_figure" -> "1234.oo"
+      )
+
+      result.status mustBe BAD_REQUEST
+    }
+
+    "redirect to CaptureLastMonthSubmitted if the box 5 figure has a correct value " in {
```
Comment (r-melvin) [inline it/uk/gov/hmrc/claimvatenrolmentfrontend/controllers/CaptureBox5FigureControllerISpec.scala:60]: Sorry, could you make this a negative value to prove that it submits correctly please?
